websites/G/Google Messages/presence.ts

```diff
@@ -6,26 +6,24 @@ tmb = Math.floor(Date.now() / 1000);
 presence.on("UpdateData", async () => {
     const presenceData: PresenceData = {
         smallImageKey: "google",
-        smallImageText: "Google"
+        smallImageText: "Google",
+        presenceData.startTimestamp = tmb;
```

Comment:
yea this doesnt work, just startTimestamp:

Reply:
KEKW